src/amqpvalue.c

```diff
@@ -6387,11 +6388,6 @@ static int internal_decoder_decode_bytes(INTERNAL_DECODER_DATA* internal_decoder
                                 {
                                     LogError("Could not create inner decoder for list items");
                                     internal_decoder_data->decoder_state = DECODER_STATE_ERROR;
-                                    result = MU_FAILURE;
```

Comment:
I wonder if valgrind will complain about this.

Reply:
not an issue